access/types/forms.py

```diff
@@ -753,3 +773,20 @@ def get_randomized_checkbox_attributes(self, i, config, initial, correct,
         random_attributes['checksum'] = self.samples_hash(random_attributes['sample'])
         return selected_choices, correct_choices, initial_choices, random_attributes
 
+
+def subdiff(value, all_solutions):
```

Comment:
I think we should change the function name `subdiff`. Maybe `get_subdiff_hints` or are there better ideas?

Reply:
The function should be renamed indeed. What about the compare method itself? 'subdiff' might be a little confusing.

Reply:
The compare method can remain `subdiff` since it describes what it does... Some kind of character differences in the string. It is supposed to be a succinct keyword anyway.